cms/djangoapps/cms_user_tasks/tasks.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from celery.task import task
+from celery.exceptions import MaxRetriesExceededError
+from boto.exception import NoAuthHandlerFound
+
+from django.core import mail
+
+log = logging.getLogger('edx.celery.task')
```

Comment:
I think most of our tasks module loggers use `get_task_logger()` to get the task name and ID automatically inserted into log messages: https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/tasks.py#L21